nbviewer/providers/local/handlers.py

```diff
@@ -53,7 +53,6 @@ def breadcrumbs(self, path):
         list
             Breadcrumbs suitable for the link_breadcrumbs() jinja macro
         """
-        provider_path = '/localfile'
```

Comment:
This isn't used. It was abstracted a few months ago, and wasn't subsequently removed.